_posts/2019-01-08-linguistics-minute-rolf-the-ganger.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Linguistics Minute: Rolf the Ganger"
+date: "2019-01-08 12:00:00 -0500"
+---
+_This was originally published in my work Slack on 2016-10-06, but I've since
+been convinced that more people than just those few who can even read it there
+might be interested. I've edited it for clarity and correctness. I'm going to
+migrate all my Linguistcs Minutes over to my blog and probably post future ones
+here._
+
+In the north of France, there's a region called Normandy. So called because it
+was "settled" by the Normans or the North Men. They were Scandinavians who
+habitually raided the southern coast of the North Sea and then sort of decided
+to not go home one time.
+
+They were led by a man named Rolf the Ganger. So, _ganger_ is a weird word,
+right? It's related to more familiar words, though: _gang_, _gangly_ and
+_gangplank_, for instance. The word _gang_ used to be a verb about walking. A
+gang is a group of people who walk together. Someone who is gangly has long
+legs, good for walking. And, well, you "walk the plank", which explains
+_gangplank_.
+
+Rolf was called "the Ganger" because he was allegedly so tall he couldn't ride a
+horse without his feet dragging on the ground so that he walked everywhere. A
+seriously big dude.
+
+There is a legend that after Rolf had settled his folks down in what is now
+Normandy, some French King (I'm not sure which one, since France wasn't actually
```

Comment:
#grammar 'King' (here and below) takes a capital when used in or as a style, e.g. 'His Majesty, King Somebody of Somewhere'; when used elsewhere, it takes a minuscule.

Reply:
Yes, yes, and no - that last one would be “the king said”, too. It’d only take the capital with the name, e.g. “King Whoozis said...”